Remotion/Data/DomainObjects.Validation.IntegrationTests/ValidationRulesIntegrationTests.cs

```diff
@@ -51,8 +51,8 @@ public void BuildValidator_MandatoryReStoreAttributeIsAppliedOnDomainObject ()
         var result1 = validator.Validate(orderItem1);
         Assert.That(result1.IsValid, Is.False);
         Assert.That(result1.Errors.Count, Is.EqualTo(1));
-        Assert.That(result1.Errors, Is.All.InstanceOf<PropertyValidationFailure>());
-        Assert.That(result1.Errors.OfType<PropertyValidationFailure>().First().ValidatedProperty.Name, Is.EqualTo("Order"));
+        Assert.That(result1.Errors, Is.All.InstanceOf<ValidationFailure>());
+        Assert.That(result1.Errors.First().ValidatedProperties.Single().Property.Name, Is.EqualTo("Order"));
```

Comment:
Contrary to the othoer comment about ValidatedProperties, this case here tests already using "First", so then using "Single" isn't an issue. 

We could (should if it's quick and easy) still update the syntax to 
```suggestion
        Assert.That(result1.Errors.First().ValidatedProperties.Select(vp => vp.Property.Name), Is.EqualTo(new [] { "Order"}));
```